src/components/HomepageBanner.js

```diff
@@ -0,0 +1,301 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { Button, Link } from '@newrelic/gatsby-theme-newrelic';
+import bannerForeground from '../images/bannerForeground.svg';
+import bannerForegroundMobile from '../images/bannerForegroundMobile.svg';
+import relicsAtWork from '../images/photo-relics-at-work-01.png';
+import relicsAtWorkMobile from '../images/relics-at-work--SM.png';
+
+const BREAKPOINTS = {
+  NO_LEFT_NAV: '760px',
+  SINGLE_COLUMN: '650px',
+  MOBILE: '450px',
+};
+
+const HomepageBanner = () => {
+  return (
+    <div
+      css={css`
+        max-height: 423px;
+        position: relative;
+        overflow: hidden;
+        border-radius: 0.5rem;
+        padding-top: calc(423 / 1252 * 100%);
```

Comment:
If we are positioning all the contents (just one div) with `position: absolute`, then we might not need to set the padding. If we're using the padding for the aspect ratio trick, then nevermind!